src/settings.js

```diff
@@ -1583,6 +1584,11 @@ var SINGLE_FILE = 0;
 // library function in library_foo.js.
 var AUTO_JS_LIBRARIES = 1;
 
+// Like AUTO_JS_LIBRARIES but for the natice libraries such as libgl, libal
+// and libhtml5.   If this is disabled it is necessary to explcitly add
+// -lhtml5 and additional to have pre-built the library using `embuilder`.
+var AUTO_LIBRARIES = 1;
```

Comment:
maybe `AUTO_NATIVE_LIBRARIES`? otherwise it is not parallel to `AUTO_JS_LIBRARIES`